package.json

```diff
@@ -18,7 +18,8 @@
     "prebuild": "yarn clean",
     "build": "yarn workspaces foreach --all --topological-dev --parallel run build",
     "test:unit": "yarn workspaces foreach --all run test:unit",
-    "test:server": "node test-server/server",
+    "dev:server": "node test-server/server",
```

Comment:
Did you rename  this for a reason? This was named `test:server` since it's a server mainly used for the test. It's not really a development server.

Reply:
Yea, I named it that way as the test prefix makes it seems like we are running a suite of tests rather than running a server.